installer/mk-install-iso.sh

```diff
@@ -51,6 +52,7 @@ rm -f ${BUILDROOT}/installer/*.pyc
 rm -rf ${BUILDROOT}/installer/BUILD_DVD
 # Copy package list json files, dereference symlinks
 cp -rf -L $PHOTON_COMMON_DIR/*.json ${BUILDROOT}/installer/
+cp -rf -L $OUTPUT_DATA_PATH/*.json ${BUILDROOT}/installer/
```

Comment:
Is that overriding the previous copying of json files? Why we need the previous line if this is the case?